modules/test/playwright/tests/address-web/countriesManagement.spec.ts

```diff
@@ -51,3 +57,50 @@ test('LPD-39651 Can activate countries in bulk', async ({
 		).toBeVisible();
 	}
 });
+
+test('LPD-41857 Can delete regions in bulk', async ({
+	apiHelpers,
+	countriesManagementPage,
+	page,
+}) => {
+	await countriesManagementPage.goto();
+
+	const regions: string[] = ['AAAA', 'BBBB', 'CCCC', 'DDDD', 'EEEE'];
+
+	for (const region of regions) {
+		await apiHelpers.headlessAdminAddress.postCountryRegion(20141, {
```

Comment:
I think we cannot hardcode 20141 because it can be different in different environment. What about getting (with API) a specific country to retrieve the id?